common/credentialFactory.go

```diff
@@ -215,6 +215,22 @@ func CreateS3Credential(ctx context.Context, credInfo CredentialInfo, options Cr
 	panic("work around the compiling, logic wouldn't reach here")
 }
 
+func CreateGCPCredential(ctx context.Context, credInfo CredentialInfo, options CredentialOpOptions) error {
```

Comment:
It looks like this is never called.  Should it be removed? Or should things be changed so it will be called?

Reply:
It wasn't necessary. Hence removed